docs/installation.rst

```diff
@@ -167,6 +167,11 @@ If you omit the keys or the endpoint URL, then Takahē will try to use implicit
 authentication for them. The keys, if included, should be urlencoded, as AWS
 secret keys commonly contain eg + characters.
 
+With the above forms, Takahē connects to an S3 bucket using **HTTPS**. If you 
```

Comment:
Could you call them "examples" rather than forms?

Reply:
done!